webapp/src/main/java/com/box/l10n/mojito/android/strings/AndroidStringDocument.java

```diff
@@ -0,0 +1,65 @@
+package com.box.l10n.mojito.android.strings;
+
+import org.w3c.dom.Node;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public class AndroidStringDocument {
+
+    private List<AndroidSingular> singulars = new LinkedList<>();
```

Comment:
will this prevent customized order, i mean intertwine singular and plurals?

Reply:
Updated